design/resources/resources-mockup.json

```diff
@@ -29,6 +29,7 @@
             "year": "2014",
             "title": "Image Resource",
             "averageRating": "1",
+            "fileName": "image.png",
```

Comment:
We have used **filename** instead of **fileName** in resources-add.component.ts
So, could you plesae change it to filename ?

Reply:
@nirojdyola Thanks for reminding me. I have changed them to filename.